src/shared/dissect-image.c

```diff
@@ -3105,28 +3116,60 @@ int dissected_image_acquire_metadata(DissectedImage *m) {
                                 r = open_extension_release(t, m->image_name, NULL, &fd);
                                 if (r < 0)
                                         fd = r; /* Propagate the error. */
-                        } else
+                                break;
+
+                        case META_HAS_INIT_SYSTEM: {
+                                bool found = false;
+                                const char *init;
+
+                                FOREACH_STRING(init,
+                                               "/usr/lib/systemd/systemd",  /* systemd on /usr merged system */
+                                               "/lib/systemd/systemd",      /* systemd on /usr non-merged systems */
+                                               "/sbin/init") {              /* traditional path the Linux kernel invokes */
```

Comment:
/usr/sbin/init?

Reply:
I am not aware of anything using /usr/sbin/init as path. The kernel default is /sbin/init, and some distros use a kernel config that calls systemd directly (and so does nspawn), but nothing uses /usr/sbin/init afaik. And I wouldn't define anything new here as entrypoint, and reuse what actually is used.

Reply:
Yeah was just thinking of getting the real path first on usrmerged systems, but it's not too important